src/bloom.h

```diff
@@ -118,6 +118,7 @@ class CRollingBloomFilter
     // A random bloom filter calls GetRand() at creation time.
     // Don't create global CRollingBloomFilter objects, as they may be
     // constructed before the randomizer is properly initialized.
```

Comment:
This comment no longer applies and can be removed

Randomness is initialized on first use